api/openapi.yml

```diff
@@ -138,6 +138,20 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  /healthcheck:
+    get:
+      summary: Health check endpoint
+      description: Health check endpoint, that run registered checkers
+      operationId: healthcheck
+    responses:
+      "200":
+        description: All checkers were in success
+      default:
+        description: Unexpected error
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/Error"
```

Comment:
I doubt we want to expose this in the public API. I'd rather expose it on the telemetry interface since it's an internal endpoint.